hack/update-codegen.sh

```diff
@@ -119,6 +114,9 @@ if [[ -z $GOPATH ]]; then
 fi
 GOPATH_1=${GOPATH%%:*}  # the first entry of the GOPATH
 
+# Enable module mode to allow setting a version in 'go get'
+export GO111MODULE=on
```

Comment:
You may also prefix the `go get` command directly, to make it obvious that this is the one being affected:

```go
( cd "$GOPATH_1" && GO111MODULE=on go get github.com/golang/mock/mockgen@v1.4.3 ) || die "Installation of mockgen failed"
```

Reply:
https://github.com/SAP/stewardci-core/pull/101/commits/7368698d9465ef9da71fd320219626f59490dae3